### ISSUE TITLE
Docs nit-picks II - The Nitpicks Strike Back

### ISSUE THREAD

**Issue by nklincoln** [hyperledger/composer#2045]
Following the latest documentation updates for Playground documentation and tutorial, we may (most likely) need a few nitpick changes. This is to contain those nit-picks.

- [x] tutorial animations are out of date:
  - step 2 (shows sample network, which is no longer there)
  - step 3 (as above)
  - step 7, animation shows the old style 'deploy' button and the wrong acl file definition
- [x] tutorial step 1 now out of date - do not deploy basic at start
- [x] tutorial step 4, model file contains back-ticks at top and bottom
- [x] tutorial step 6, acl file contains back-ticks
- [x] tutorial step 7, we test the deployed business network
- [x] tutorial step 9, items contain back-ticks
- [x] tutorial step 10, back-ticks
- [x] tutorial step 10, item 3 ... after creating this asset, you should be able to see *if* in the (should be 'it')
- [x] tutorial step 11, back-ticks
- [x] tutorial step 11, item 6, the button name needs capitalisation

**Comment by mbwhite** [comment]
Edge & IE

- [ ] SVG images are black blobs
- [ ] All the documentation pages display navigation full width at the top and the contents below that

**Comment by KaiUsher** [comment]
Step Four: Adding a model file - code to copy contains ``` at top and bottom.

![screen shot 2017-08-31 at 10 36 57](https://user-images.githubusercontent.com/16639383/29917026-856799f6-8e38-11e7-8578-4648ff938e2a.png)

**Comment by jt-nti** [comment]
Probably just me, but there seem to be a lot of _check our documentation_ links in the tutorial. (It's probably not great for anyone tabbing through links with a screen reader.) For example, rather than...

For more information on access control files, check our [documentation](https://hyperledger.github.io/composer/unstable/reference/acl_language.html).

I would personally prefer something like

Check our [access control files documentation](https://hyperledger.github.io/composer/unstable/reference/acl_language.html) for more information.

**Comment by jt-nti** [comment]
Step seven screen animation does not show the ACL that was just pasted in...

![image](https://user-images.githubusercontent.com/3535067/29917860-2d0c0604-8e3b-11e7-8f66-5d780ab06881.png)

**Comment by mbwhite** [comment]
- [ ] Playground tutorial does not tell to press update (not that it should because the playground should keep the contents of the editors)

**Comment by jt-nti** [comment]
Text does not match screen animation in step nine. (Just swap them round?)

![image](https://user-images.githubusercontent.com/3535067/29918225-9a964b3e-8e3c-11e7-9ccf-3e210759fcc3.png)

**Comment by RJackson1234** [comment]
- [ ]  (1) If you are new we say go to the Playground tutorial ... if not we say follow these instructions which implies that the first thing you need to do is start dealing with ID cards which we've already said is an Advanced feature. Still not sure the order is right here.

- [ ]  (2) Landing Page

The Wallet page is the default Playground landing page    ... Why do we need to talk about a 'landing page' ... let's just describe 'The Wallet' and not introduce another term.

- [ ]  (3)  Once connected to a deployed Business Network,...but we haven't said how to deploy a business network at this stage?

- [ ]  (4)  Deploy new business network ‘card’. If this is your first time using Playground, or you wish to start a new network, you should press this button ..... is this really a button? (extreme nitpick!!)

- [ ] (5) The card provides options to delete the identity/card.... can you delete one without the other? 

- [ ]  (6) simply browse to the file ..... spurious 'to'

- [ ] (7) Connect using credentials .... what are credentials? What do they look like? Where would you get them from? How can I check if I have any?

- [ ] (8) Editor Global Options ... what are these? Why are we calling them Global Options? I'm assuming Define and Test are the only ones we have?

- [ ] (9) The Editor define and test pages contain a 'header that shares interaction options' .... not sure what this actually means. I think we're trying to say there are 2 editing options .. Define and Test but are bringing in 'header' and 'interaction' terms which is confusing?

- [ ] (10) A drop-down that is labelled with the current Identity... we should say (Admin) here

- [ ] (11) Identity Registry (for issuing identities)  - think this needs more explanation. It's the first time we've mentioned this concept. 

- [ ]  (12) Editor Define page is the page  ... Can we not just say the 'Define' page?

- [ ]  (13) Add File button ... this doesn't look like a button ... The Update button does.. why is this different?

- [ ]  (14) Editor Test page .. Can we not just say the 'Test' page.. not sure we need to expose it as an Editor?

- [ ]  (15) Historian Records ... do we need to expose the name Historian? are they not just records?

- [ ]  (16) sumbission of a Transaction ... spelling (-:

- [ ]  (17) ID Card .....using the icons present on the card...can we add the icons into the text?

- [ ]  (18) An ID Card can be created when an Identity is issued .... we haven't explained what an Identity is or how it could be issued or where it comes from?

- [ ]  (19) If a user is provided with a Enrollment ID and Secret... what do these look like? Where would I find them?

- [ ]  (20) which corresponds to a valid Identity  ... how do you know if it's valid? How can you check this? Where would the user look?

- [ ] (21) It is also possible to manually create an ID Card..... how do I do this?

- [ ]  (22) There must be a Participant existing in the business network in order to bind an Identity required to create the ID Card.... Are we saying that you have to first create a participant before you can issue an ID card? How do I do a 'bind'?

- [ ]  (23) The ID Card will appear in your wallet once you log out of the current business network.. so are we saying the only way to check you have created one is to log out? (That doesn't seem great?)

- [ ] (24) Creating an ID Card ... can we slow the MP4 down ... it's hard to follow?

- [ ] (25) Please note: You must use an identity that has permission to create new identities.... how can I check what permissions an identity has?

- [ ]  (26) +Issue New ID. ... why have we added the '+' to this?

**Comment by EdProsser** [comment]
I'm gonna shelve most of the comments from @RJackson1234 for another set of changes, I've got all the stop-ships done.

**Comment by EdProsser** [comment]
Ok, stuff from Dan on Friday: 
[ ] Queries and ID cards going in list of concepts.
[ ] Remove line about GitHub account in getting-started-with-playground.md
